xchange-stream-hitbtc/src/main/java/info/bitrich/xchangestream/hitbtc/dto/HitbtcOrderBookWithTimestamp.java

```diff
@@ -0,0 +1,40 @@
+package info.bitrich.xchangestream.hitbtc.dto;
```

Comment:
I can't see how this is being used.  I'd expect to see a corresponding change in https://github.com/bitrich-info/xchange-stream/blob/develop/xchange-stream-hitbtc/src/main/java/info/bitrich/xchangestream/hitbtc/HitbtcStreamingMarketDataService.java to feed the timestamp to clients?

Reply:
Locally, I extended _HitbtcAdapters.adaptOrderBook_ method from the _org.knowm.xchange_ library to support timestamps. However, ideally, it should be updated at the source library, rather than "overridden" here. Having said that, I'm fine to commit it as well if it does make sense.